mailbox/store/src/main/java/org/apache/james/mailbox/store/event/distributed/RegisteredDelegatingMailboxListener.java

```diff
@@ -25,7 +25,9 @@
 import org.apache.james.mailbox.MailboxListener;
```

Comment:
Hint: MailboxPathRegister should become a MailboxIdRegister ;-)

Reply:
I won't touch on that because it's related to database structure for mailboxPath and its topic